### ISSUE TITLE
Cyrillic + supplement 

### ISSUE THREAD

**Issue by typemytype** [LettError/glyphNameFormatter#32]
some conflicts in Cyrillic supplement range 0500..052F

both

```
CYRILLIC CAPITAL LETTER EL WITH MIDDLE HOOK
CYRILLIC CAPITAL LETTER EL WITH HOOK
```

will process to Elhookcyr

also 

```
Enhookcyr                               Cyrillic                                CYRILLIC CAPITAL LETTER EN WITH HOOK
Enhookcyr                               Cyrillic Supplement                     CYRILLIC CAPITAL LETTER EN WITH MIDDLE HOOK
```

Changing MIDDLE HOOK will affect (in Cyrillic range 0400..04FF)

```
cy-Ghemiddlehook    Ghemiddlehook   Gehook  0494    Ҕ  CYRILLIC CAPITAL LETTER GHE WITH MIDDLE HOOK
cy-ghemiddlehook    ghemiddlehook   gehook  0495    ҕ  CYRILLIC SMALL LETTER GHE WITH MIDDLE HOOK

cy-Pemiddlehook Pemiddlehook    Pehook  04A6    Ҧ  CYRILLIC CAPITAL LETTER PE WITH MIDDLE HOOK
cy-pemiddlehook pemiddlehook    pehook  04A7    ҧ  CYRILLIC SMALL LETTER PE WITH MIDDLE HOOK
```

**Comment by LettError** [comment]
Is there an objection against `cy-Ghemiddlehook` ?

**Comment by twardoch** [comment]
`*midhook` or `*middlehook` is better for all those letters except U+0512 (CYRILLIC CAPITAL LETTER EL WITH HOOK). The middlehook really isn’t a "hook" comparable to what you have in Vietnamese. So I'd prefer middlehook in all those.

**Comment by LettError** [comment]
Fixed in #60.

```Elmiddlehookcyr 0520
elmiddlehookcyr 0521
Enmiddlehookcyr 0522
enmiddlehookcyr 0523
```
